src/app/header/header.component.html

```diff
@@ -98,6 +98,7 @@ <h2 class="header_descr">
             {{roles[(user$ | async).role]}}</button></a>
         <a [routerLink]="'./personal-cabinet/workshops'"><button mat-menu-item>Мої гуртки</button></a>
         <a [routerLink]="'./personal-cabinet/applications'"><button mat-menu-item>Мої заяви</button></a>
+        <a *ngIf="(user$ | async).role === Role.provider" [routerLink]="'./personal-cabinet/administration/all'"><button mat-menu-item>Адміністрування</button></a>
```

Comment:
I can see that we have too many subscriptions, I believe it can have single subscription which sets variable

Reply:
it has been replaced with variable now